modules/library/render/src/main/java/org/geotools/renderer/style/DynamicSymbolFactoryFinder.java

```diff
@@ -47,6 +50,9 @@
     protected static final Logger LOGGER =
             org.geotools.util.logging.Logging.getLogger(DynamicSymbolFactoryFinder.class);
 
+    public static final Hints.Key MARK_FACTORY_ORDER = new Hints.Key(Comparator.class);
```

Comment:
Bit of javadoc for the new public fields?

Reply:
Makes sense, adding them.